### ISSUE TITLE
Remove last_login from pg-people

### ISSUE THREAD

**Issue by mk4111** [emfoundation/ce100-app#586]
We currently have the field `last_login` in the people plugin. This is not used/updated in our app, when we activate an account, log in, etc. It does however exist in our tests.

Given that this is currently not used, I think it would make most sense if it was removed for the time being, so as not to get confusde by it's existence...

**Comment by JMurphyWeb** [comment]
to clarify this is to remove the values from mock data (`last_login: null`) or to remove the keys altogether?

**Comment by mk4111** [comment]
remove the keys altogether methinks! But what do you think @JMurphyWeb ? :cake:

**Comment by JMurphyWeb** [comment]
yeah could do. Or for now just make them all `null` in the mock data?

@asimonw can you think of a reason to keep last login in the data structure?

**Comment by asimonw** [comment]
@JMurphyWeb We do often find that useful more from an internal analytics point of view. It might also be useful when more advanced features get introduced that depend on whether the person hasn't logged in for a while or not. In general last login is a useful property on a user model, I think. Maybe check with @nelsonic.

**Comment by JMurphyWeb** [comment]
Sounds good. I guess the actionable task here is to remove any mock last login data as it would possibly confuse any dev coming into the project. 

Then when the need requires, we can actually start recording last-login when a user logs in.
